### PR TITLE
fix: jira verify enckey and jira encode password

### PR DESCRIPTION
**Pull request by abeizn**
# Summary

jira verify enckey and jira encode password

### Does this close any open issues?
fix: #2529 

### Screenshots
Include any relevant screenshots here.

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: plugins/jira/models/migrationscripts/init_schema.go [hunk 1]
```diff
@@ -131,6 +138,9 @@ func (*InitSchemas) Up(ctx context.Context, db *gorm.DB) error {
 
 				c := config.GetConfig()
 				encKey := c.GetString("ENCODE_KEY")
+				if encKey == "" {
+					return fmt.Errorf("jia v0.10 invalid encKey")
```
Comment (likyh) [inline plugins/jira/models/migrationscripts/init_schema.go:142]: typo

File: plugins/jira/models/migrationscripts/updataSchemas20220505.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
+/*
```
Comment (warren830) [inline plugins/jira/models/migrationscripts/updataSchemas20220505.go:1]: please use update

File: plugins/jira/models/migrationscripts/updataSchemas20220505.go [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+func (*UpdateSchemas20220505) Version() uint64 {
+	return 20220505212344
+}
+
+func (*UpdateSchemas20220505) Owner() string {
+	return "Jira"
+}
+
+func (*UpdateSchemas20220505) Name() string {
+	return "Rename source to connection "
```
Comment (warren830) [inline plugins/jira/models/migrationscripts/updataSchemas20220505.go]: how about changing to preparation for init_schemas

**Review by warren830 — CHANGES_REQUESTED** [review]

File: plugins/jira/models/migrationscripts/updataSchemas20220505.go (hunk 2 above)
Comment (abeizn) [reply plugins/jira/models/migrationscripts/updataSchemas20220505.go]: updated

File: plugins/jira/models/migrationscripts/updataSchemas20220505.go (hunk 3 above)
Comment (abeizn) [reply plugins/jira/models/migrationscripts/updataSchemas20220505.go]: updated

**Review by warren830 — APPROVED** [review]
LGTM

**Pull request merged**